app/services/vector_service.py

```
@@ -0,0 +1,294 @@
+            session=session,
+            user_id=user_id,
+            source_id=source_id,
+            source_type=source_type,
+            chunks=chunks,
+            embeddings=embeddings,
+            title=title,
+            url=url,
+            keywords=keywords,
+            summary=summary,
+            embedding_model=self.embeddings.model,
+            chunk_size=self.text_splitter._chunk_size,
```

Pylint (3.3.7)

[warning] Access to a protected member _chunk_size of a client class

(W0212)